.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint and Test
+
+on:
+  push:
+    branches: 
+      - develop
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches: 
+      - develop
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: yarn install, lint and build
+      run: |
+        yarn install --frozen-lockfile
+        yarn lint
+        # tests are not fully supported atm
+        # yarn testCI
```

Comment:
How/When are tests run now?

Reply:
tests for the demo client did never run. Most of the tests are also not working. If we want to have tests here, we would do that in another ticket